cmake/version.cmake

```
@@ -62,7 +62,7 @@ endif()
 set(PROJECT_VERSION_STR ${PROJECT_VERSION}${PROJECT_VERSION_EXTRA_STR})
 
 if (NOT NO_PRINT_VERSION)
-  message(STATUS "Zephyr version: ${PROJECT_VERSION_STR}")
+  message(STATUS "Zephyr version: ${PROJECT_VERSION_STR} (${ZEPHYR_BASE})")
```

Not against also printing here, but I think we should then inform what we are printing.
```suggestion
  message(STATUS "Zephyr version: ${PROJECT_VERSION_STR} (ZEPHYR_BASE: ${ZEPHYR_BASE})")
```

but without this patch we would still print (first run):
```
Including boilerplate (in-zephyr-tree): /projects/github/zephyr-upstream/zephyr/cmake/app/boilerplate.cmake
-- Zephyr version: 2.2.0-rc2
```
so if you look at first line, it prints the boilerplate, and the boilerplate is included from the current ZEPHYR_BASE, so you already have this information available.

Subsequent runs would print:
```
Including boilerplate (zephyr base (cached)): /projects/github/zephyr-upstream/zephyr/cmake/app/boilerplate.cmake
-- Zephyr version: 2.2.0-rc2
```

---

squashed in your suggested change.
I believe it is nice to explicitly inform users of Zephyr base, although it is indirectly stated through boilerplate inclusion.